README.md

```diff
@@ -1,10 +1,10 @@
 # nichtparasoup
 
 [![PyPI](https://img.shields.io/pypi/v/nichtparasoup)](https://pypi.org/project/nichtparasoup/)
-![PyPI - Python Version](https://img.shields.io/pypi/pyversions/nichtparasoup)  
```

Comment:
could you revert the removal of the trailing spaces.
they were intended for a line break